lesson11/client_src/components/bar/bar.spec.js

```diff
@@ -0,0 +1,22 @@
+var Bar  = require('./bar.js');
+var Resource  = require('../../models/resource.js');
```

Comment:
You need to remove `Resource` dependency.  You should test only one "unit" and this should be `Bar`